### DISCUSSION TITLE
Problems with importModel()

### DISCUSSION THREAD

**Discussion by JBiaggi** [SysBioChalmers/RAVEN#561] [Help me!]
Hello! I recently reinstalled RAVEN using MATLAB (2024b) Add-Ons. Ever since I'm having this problem with importModel(): sometimes it works, sometimes it doesn't. Usually what happens is: when I run the code for the first time, I get this error message:

`importModel('iYali.xml')
No model returned.

Error using dispEM (line 49)
There is a problem with the SBML file. Try using the SBML Validator at http://sbml.org/Facilities/Validator

Error in importModel (line 136)
    dispEM(EM);`

Then I run the code again, and the function works! The SBML Validators reports that everything is fine with the model. Could you help me fix this problem somehow?

Kind regards

**Comment by edkerk** [comment]
I'll transfer this to an Issue as it seems to be a consistent problem:_ #562.
